ams/server/src/main/java/com/netease/arctic/server/optimizing/plan/OptimizingPlanner.java

```diff
@@ -126,10 +128,10 @@ public List<TaskDescriptor> planTasks() {
     evaluators.sort(Comparator.comparing(PartitionEvaluator::getWeight, Comparator.reverseOrder()));
 
     double maxInputSize = MAX_INPUT_FILE_SIZE_PER_THREAD * availableCore;
-    List<PartitionEvaluator> inputPartitions = Lists.newArrayList();
+    actualPartitionPlans = Lists.newArrayList();
```

Comment:
This place is a bit strange. Perhaps we can directly return fromSequence and toSequence as the result?

Reply:
This requires us to make refactors to the interface, which may not be suitable to complete in this hotfix.